packages/pointer_interceptor/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
-## 0.9.2
+## 0.9.3
 
 * Marked `PointerInterceptor` as invisible, so it can be optimized by the engine.
+* Require minimal version of flutter SDK to be `2.10`
+
+## 0.9.2
```

Comment:
Looks like you added trailing whitespace here.

Reply:
I looked for it everywhere, but couldn't find it :P